#! /bin/sh -e
## 200_de-manpages.dpatch by <unknown>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: German man pages corrections/additions

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0

@DPATCH@

diff -Nru shadow-4.0.3.ori/man/de/Makefile.am shadow-4.0.3_31/man/de/Makefile.am
--- shadow-4.0.3.ori/man/de/Makefile.am	2002-03-13 19:44:10.000000000 +0100
+++ shadow-4.0.3_31/man/de/Makefile.am	2005-01-07 18:48:37.000000000 +0100
@@ -3,6 +3,8 @@
 
 man_MANS = \
 	chsh.1 \
+	vipw.8 \
+	vigr.8 \
 	passwd.1
 
 EXTRA_DIST = $(man_MANS)
+	install-strip installcheck installcheck-am installdirs \
+	maintainer-clean maintainer-clean-generic mostlyclean \
+	mostlyclean-generic mostlyclean-libtool pdf pdf-am ps ps-am \
+	uninstall uninstall-am uninstall-info-am uninstall-man \
+	uninstall-man1 uninstall-man8
 
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
diff -Nru shadow-4.0.3.ori/man/de/vigr.8 shadow-4.0.3_31/man/de/vigr.8
--- shadow-4.0.3.ori/man/de/vigr.8	1970-01-01 01:00:00.000000000 +0100
+++ shadow-4.0.3_31/man/de/vigr.8	2005-01-07 18:48:37.000000000 +0100
@@ -0,0 +1 @@
+.so man8/vipw.8
diff -Nru shadow-4.0.3.ori/man/de/vipw.8 shadow-4.0.3_31/man/de/vipw.8
--- shadow-4.0.3.ori/man/de/vipw.8	1970-01-01 01:00:00.000000000 +0100
+++ shadow-4.0.3_31/man/de/vipw.8	2005-01-07 18:48:37.000000000 +0100
@@ -0,0 +1,32 @@
+.\"$Id$
+.\"
+.\"Translated into german by Helge Kreutzmann <kreutzm@itp.uni-hannover.de>
+.TH VIPW 8 "21 Jul 2004"
+.SH NAME
+vipw, vigr \- bearbeitet die Passwort-, Gruppen, Shadow-Paßwort oder Shadow-Gruppen-Datei.
+.SH SYNTAX
+\fBvipw\fR [\fB-s\fR]
+.br
+\fBvigr\fR [\fB-s\fR]
+.SH BESCHREIBUNG
+.BR vipw " und " vigr
+bearbeiten die Dateien
+.IR /etc/passwd " bzw. " /etc/group .
+Mit dem 
+.B -s
+Schalter bearbeiten sie die Shadow-Version dieser Dateien,
+.IR /etc/shadow " bzw. " /etc/gshadow .
+Diese Programme setzen die entsprechenden Locks um Dateibeschädigung
+zu verhindern.
+
+Bei der Auswahl des Editors überprüfen die Programme zuerst
+die Umgebungsvariable
+.BR VISUAL ,
+dann die Umgebungsvariable
+.BR EDITOR ,
+und schließlich den Aufruf des Standard-Editors
+.BR vi (1).
+.SH "SIEHE AUCH"
+.BR group (5),
+.BR passwd (5),
+.BR shadow (5)
